es-models/case_set/case_set.mapping.yaml

```diff
@@ -0,0 +1,10 @@
+_all:
+  enabled: false
+properties:
+  case_ids:
```

Comment:
don't think this needs to be nested - since it is just the one field you can make it `keyword`. ES doesn't care about array types.